apps/landingPage/config/GA.js

```diff
@@ -0,0 +1,76 @@
+// @flow
+
+import ReactGA from 'react-ga';
+
+export const GOOGLE_ANALYTICS_TRACKING_ID = 'UA-29345084-41';
+
+export const EventLandingPageReadMore = () => {
+  ReactGA.event({
```

Comment:
Not sure if we need to track it.

Reply:
I think it's important. We want to know what is more attractive for users (github or read more). We don't have any other way how to do it

Reply:
Alright :)